spec/em_spec.rb

```diff
@@ -0,0 +1,63 @@
+#spec/em_spec.rb
+require './lib/enumerable-methods'  #=> add this
+
+
+
+RSpec.describe Enumerable do
```

Comment:
Kindly put all tests within `RSpec.describe Enumerable do` block, you don't need to write it for each test